isthmus/src/test/java/io/substrait/isthmus/Substrait2SqlTestNoValidation.java

```diff
@@ -0,0 +1,45 @@
+package io.substrait.isthmus;
+
+import static io.substrait.isthmus.SqlToSubstrait.EXTENSION_COLLECTION;
+
+import io.substrait.relation.Rel;
+import java.util.Arrays;
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.rel.RelRoot;
+import org.apache.calcite.sql.SqlDialect;
+import org.junit.jupiter.api.Test;
+
+public class Substrait2SqlTestNoValidation extends PlanTestBase {
+  @Test
+  public void simpleTest() throws Exception {
+    String query = "select p_size + 100 from part where p_partkey > 100";
```

Comment:
let's add some tests that are sql => substrait => calcite plan => substrait and then do comparison of the substrait plans (round trips).

Reply:
Add the sql => substrait => calcite plan => substrait in `Substrait2SqlTest`. 

For query with implicit cast involving literal, such as l_partkey > 100,  the first substrait plan will contain a CAST(100 as bigint, the second substrait plan will have 100 as i64 literal.  I have not located the exact places where the CAST is removed.  So, for now, the testcases do not contain cases with implicit cast over literals. 

```
Cast{type=I64{nullable=false}, input=I32Literal{nullable=false, value=100}

I64Literal{nullable=false, value=100}], outputType=Bool{nullable=false}
```